Fix ce_switchport module with connection type ansible.netcommon.netconf

efrikin opened the pull request:
##### SUMMARY
<!--- Describe the change below, including rationale and design decisions -->
Unused parameter for `ce_switchport` module always broken launch

Fixes #313 and #137
<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->

##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
- Bugfix Pull Request

##### COMPONENT NAME
<!--- Write the short name of the module, plugin, task or feature below -->
plugins/modules/network/cloudengine/ce_switchport.py

##### ADDITIONAL INFORMATION
[Reproduce](https://github.com/ansible-collections/community.network/issues/313)

efrikin commented:
/rebuild

efrikin commented:
+label stale_ci

felixfontein commented:
I don't really know the module, but your fix looks ok to me, as these properties aren't used anywhere. In any case, you need a changelog fragment (https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to).

efrikin commented:
> I don't really know the module, but your fix looks ok to me, as these properties aren't used anywhere. In any case, you need a changelog fragment (https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to).

@felixfontein hello

Thank a lot the answer.

I added changelog fragment file and updated example inside `ce_switchport.py`.

dericcrago commented:
thanks @efrikin & @felixfontein !
